### PR TITLE
[14.0] account_asset_management: improve and cleanup views

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Review by kittiu — APPROVED** [review]

File: account_asset_management/views/account_move.xml [hunk 1]
```diff
@@ -45,8 +45,13 @@
                 <field
                     name="asset_profile_id"
                     domain="[('company_id','=', parent.company_id)]"
+                    optional="show"
```
Comment (pedrobaeza) [inline account_asset_management/views/account_move.xml]: I think this one should be `hide`. Most of the time you are not entering asset invoices.

File: account_asset_management/views/account_asset.xml [hunk 2]
```diff
@@ -334,28 +322,30 @@
                     string="Draft"
                     name="draft"
                     domain="[('state', '=', 'draft')]"
-                    help="Draft Assets"
                 />
                 <filter
                     string="Running"
                     name="running"
                     domain="[('state', '=', 'open')]"
-                    help="Assets in Running State"
```
Comment (pedrobaeza) [inline account_asset_management/views/account_asset.xml:343]: Why removing this? Isn't working?

Comment (alexis-via) [reply account_asset_management/views/account_asset.xml:343]: It brings 0 additionnal information. It just gives more work for translators for nothing...

File: account_asset_management/views/account_move.xml (hunk 1 above)
Comment (alexis-via) [reply account_asset_management/views/account_move.xml]: ok, done

File: account_asset_management/views/account_asset.xml (hunk 2 above)
Comment (pedrobaeza) [reply account_asset_management/views/account_asset.xml:343]: OK then

**Pull request merged**